package.json

```diff
@@ -69,7 +69,7 @@
   },
   "dependencies": {
     "@apollographql/graphql-playground-html": "^1.6.24",
-    "accept": "^3.1.3",
+    "@hapi/accept": "^5.0.0",
```

Comment:
I don't think you can jump all the way to version 5 of this module.  Reading through the changelog, version 5 was a breaking change that altered support to ONLY version 12 of node.  That seems like a strange decision since node 10 is still under active support.

With that said, I don't think anything in the moleculer ecosystem should require node 12.  It looks like @hapi/accept 4.0.0 is the last version which doesn't require node 12.

Reply:
@shawnmcknight, let me check once, coz that I'm using node version `10.16.1`

Reply:
@shawnmcknight Checking the code, can see that they mentioned ONLY version 12 means they test in v12 only. Not meaning don't support older version. Anyway, I think, it could be better to keep `@hapi/accept@4.0.0`. Everything still working well.